src/hooks/useHasAppUpdated/useHasAppUpdated.ts

```diff
@@ -7,6 +7,8 @@ const APP_UPDATE_CHECK_INTERVAL = 1000 * 60
 export const useHasAppUpdated = () => {
   const [hasUpdated, setHasUpdated] = useState(false)
   useInterval(async () => {
+    // we don't care about updates locally obv
+    if (window.location.hostname === 'localhost') return
```

Comment:
introducing a dependency on `window.location.hostname` seems brittle to me; how about checking an env var?
```suggestion
    if (process.env.NODE_ENV !== 'production') return
```